watertap/tools/parameter_sweep/parameter_sweep.py

```diff
@@ -504,32 +613,48 @@ def _create_global_output(self, local_output_dict, req_num_samples=None):
 
         return global_output_dict
 
-    def _param_sweep_kernel(self, model, reinitialize_values):
+    def _param_sweep_kernel(self, sweep_params, local_value_k):
         optimize_function = self.config.optimize_function
         optimize_kwargs = self.config.optimize_kwargs
-        reinitialize_before_sweep = self.config.reinitialize_before_sweep
-        reinitialize_function = self.config.reinitialize_function
-        reinitialize_kwargs = self.config.reinitialize_kwargs
-
+        build_model = self.config.build_model
+        build_model_kwargs = self.config.build_model_kwargs
+        initialize_function = self.config.initialize_function
+        initialize_kwargs = self.config.initialize_kwargs
+        initialize_before_sweep = self.config.initialize_before_sweep
+        update_sweep_params_before_init = self.config.update_sweep_params_before_init
+        # Depreciated
+        # reinitialize_function = self.config.reinitialize_function
+        # reinitialize_kwargs = self.config.reinitialize_kwargs
```

Comment:
The let's deprecate it -- remove these commented out lines and these options from the Config object.